src/pages/workspaces/workspace/analysis/ContextBar.js

```diff
@@ -147,7 +147,7 @@ export const ContextBar = ({
     const curPd = getCurrentPersistentDisk(runtimes, persistentDisks)
     const diskCost = curPd ? getPersistentDiskCostHourly(curPd, computeRegion) : 0
     const display = Utils.formatUSD(galaxyRuntimeCost + galaxyDiskCost + runtimeCost + diskCost)
-    return `${display}`
+    return !isGoogleWorkspace(workspace) && currentRuntime && currentRuntime.status === 'Stopped' ? '$0.00' : `${display}` //[IA-4105] TODO: add azure cost calculation
```

Comment:
this logic should not be in the context bar

Reply:
I think we need to add this logic into `getRuntimeCost`! This method already has an Azure case. A check for status === 'Stopped' could be added into that to return $0 instead of calling `getAzureComputeCostEstimate`.